common/config/rush/pnpm-lock.yaml

```diff
@@ -103,13 +103,14 @@ packages:
     dev: false
     resolution:
       integrity: sha512-pkFCw6OiJrpR+aH1VQe6DYm3fK2KWCC5Jf3m/Pv1RxF08M1Xm08RCyQ5Qe0YyW5L16yYT2nnV48krVhYZ6SGFA==
-  /@azure/eslint-plugin-azure-sdk/2.0.1_ee45056a3a53530214d828c032d67185:
+  /@azure/eslint-plugin-azure-sdk/2.0.1_5be9065a4c7972ebfd372404cdf76b9c:
     dependencies:
-      '@typescript-eslint/parser': 2.23.0_eslint@6.8.0+typescript@3.7.5
+      '@typescript-eslint/parser': 2.25.0_eslint@6.8.0+typescript@3.7.5
       eslint: 6.8.0
       fast-levenshtein: 2.0.6
       glob: 7.1.6
       typescript: 3.7.5
+    deprecated: 'This package is now a private implementation detail of https://github.com/Azure/azure-sdk-for-js'
```

Comment:
Unrelated to `rush update --full`, but should we still have reference to this package now that the sources live in the repo?

Reply:
we eventually need to update to the version 3.0.0 (the eslint-plugin-sdk in our repo) but updating that requires some code changes. so @willmtemple will be doing the changes outside of this PR